diff --git a/node_modules/react-virtualized/dist/commonjs/Table/defaultCellDataGetter.js b/node_modules/react-virtualized/dist/commonjs/Table/defaultCellDataGetter.js
index cc71553..f728d75 100644
--- a/node_modules/react-virtualized/dist/commonjs/Table/defaultCellDataGetter.js
+++ b/node_modules/react-virtualized/dist/commonjs/Table/defaultCellDataGetter.js
@@ -15,7 +15,7 @@ var _types = require("./types");
 function defaultCellDataGetter(_ref) {
   var dataKey = _ref.dataKey,
       rowData = _ref.rowData;
-
+  if(rowData == undefined) return ''; //hack patch -- coded by wenhao.xie
   if (typeof rowData.get === 'function') {
     return rowData.get(dataKey);
   } else {
diff --git a/node_modules/react-virtualized/dist/es/Table/defaultCellDataGetter.js b/node_modules/react-virtualized/dist/es/Table/defaultCellDataGetter.js
index 0bb93c4..c1fb431 100644
--- a/node_modules/react-virtualized/dist/es/Table/defaultCellDataGetter.js
+++ b/node_modules/react-virtualized/dist/es/Table/defaultCellDataGetter.js
@@ -6,7 +6,7 @@
 export default function defaultCellDataGetter(_ref) {
   var dataKey = _ref.dataKey,
       rowData = _ref.rowData;
-
+  if(rowData == undefined) return ''; //hack patch -- coded by wenhao.xie
   if (typeof rowData.get === 'function') {
     return rowData.get(dataKey);
   } else {
